modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/operation/OperationDefinitionParser.java

```diff
@@ -30,6 +40,31 @@ public OperationDefinitionParser(Builder definition, ExtensionModel extensionMod
     super(definition, extensionModel, operationModel, dslSyntaxResolver, parsingContext);
   }
 
+  @Override
+  protected Builder doParse(Builder definitionBuilder) throws ConfigurationException {
```

Comment:
why this? what if we ever wanted to add this capability to a construct or a source?